packs/feats/deathless-servant.json

```diff
@@ -28,11 +28,18 @@
             "remaster": true,
             "title": "Pathfinder War of Immortals"
         },
-        "rules": [],
+        "rules": [
```

Comment:
Consider adding the healing to the description, and adding an RE for the status bonus to damage.